tests/integration/Dockerfile.girder

```diff
@@ -0,0 +1,29 @@
+FROM girder/girder:latest
+MAINTAINER Christopher Kotfila <chris.kotfila@kitware.com>
+
+VOLUME /girder/plugins/integration_test_endpoints
+
+RUN sed -i 's|^uri = "mongodb://localhost:27017/girder"$|uri = "mongodb://mongo:27017/girder"|' /girder/girder/conf/girder.local.cfg
+RUN sed -i 's|^server.socket_port = 8080$|server.socket_port = 8989|' /girder/girder/conf/girder.local.cfg
+
+RUN pip install ansible girder-client
+
+COPY ./scripts /scripts
+RUN mkdir -p /scripts/roles && ansible-galaxy install --roles-path /scripts/roles girder.girder
+
+
+COPY ./integration_test_endpoints /girder/girder/plugins/
+RUN girder-install web --all-plugins
+
+# Allow mounting integration_test_endpoints from host environment
+# e.g. for developing test endpoints
+VOLUME /girder/girder/plugins/integration_test_endpoints
+
+# Allow moutning /girder_worker
+RUN mkdir /girder_worker
+VOLUME /girder_worker
+
+# Make sure remote debugging is available
+RUN pip install rpdb
+
```

Comment:
Should the logs be mounted somehow? I can imagine after the tests we would want to place Girder error/info logs into Circle artifacts.